iree/compiler/Dialect/HAL/Target/LLVM/LLVMTargetOptions.cpp

```diff
@@ -107,6 +107,13 @@ LLVMTargetOptions getLLVMTargetOptionsFromFlags() {
       llvm::cl::init(llvmTargetOptions.debugSymbols));
   llvmTargetOptions.debugSymbols = clDebugSymbols;
 
+  static llvm::cl::opt<bool> clAddressSanitizer(
+      "iree-llvm-address-sanitizer",
+      llvm::cl::desc("Turn on address sanitization to detect memory violations"),
+      llvm::cl::init(llvmTargetOptions.addressSanitizer));
+//  if(clAddressSanitizer)
```

Comment:
Please self-review before sending PRs to make sure you don't leave stray whitespace changes and comments :)